### DISCUSSION TITLE
everything调用当前目录搜索

### DISCUSSION THREAD

**Discussion by carlosyou** [shuax/MouseInc.Settings#181] [General]
你好！
想设置一个快捷键调用everything搜索当前目录，请问“当前目录”的参数怎么填写？
目前已实现：
快捷键：Ctrl+Alt+Q  已可直接呼出everyth软件界面
RAW：
[
	["Execute", "\"D:\\Green Apps\\Everything\\Everything.exe\" -p \"XXXX""]
]
其中：XXXX 还请指教，谢谢！
例如：资源管理器停留在F:\123，希望打开everything后能直接搜索F:\123下的内容，而不是全盘
之前尝试过：%path% ，但调用的是WINDOWS系统的环境变量

**Comment by shuax** [comment]
目前获取不到当前目录
